opsdroid/constraints.py

```diff
@@ -31,7 +31,7 @@ def constraint_decorator(func):
 
         def constraint_callback(message, rooms=rooms):
             """Check if the room is correct."""
-            return message.target in rooms
+            return message.target in lookup_target(message, rooms)
```

Comment:
This line would then become
```suggestion
            return message.target in message.connector.lookup_target(targets)
```

Although I feel like the connector method should be singular, i.e. it tries to convert a human-friendly name to an ID. So maybe something like this:
```suggestion
            return message.target in list(map(message.connector.lookup_target, rooms))
```
(Where `lookup_target` takes one target alias only)